### ISSUE TITLE
CMake Error at CMakeLists.txt:85 (message): gperf not found

### ISSUE THREAD

**Issue by skupr-anaconda** [FreeTDS/freetds#396]
When freetds was built on win platform there was an error:
```
CMake Error at CMakeLists.txt:85 (message):
  gperf not found!

```
But gperf is the package for unix not win platforms.

Logs:

```
    active environment : base
    active env location : C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env
            shell level : 1
       user config file : C:\Users\builder\.condarc
 populated config files : C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env\.condarc
          conda version : 4.10.0
    conda-build version : 3.21.4
         python version : 3.8.8.final.0
       virtual packages : __cuda=10.1=0
                          __win=0=0
                          __archspec=1=x86_64
       base environment : C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env  (writable)
      conda av data dir : C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env\etc\conda
  conda av metadata url : https://repo.anaconda.com/pkgs/msys2
           channel URLs : https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
          package cache : C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env\pkgs
                          C:\Users\builder\.conda\pkgs
                          C:\Users\builder\AppData\Local\conda\conda\pkgs
       envs directories : C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env\envs
                          C:\Users\builder\.conda\envs
                          C:\Users\builder\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/4.10.0 requests/2.25.1 CPython/3.8.8 Windows/10 Windows/10.0.17763
          administrator : True
             netrc file : None
           offline mode : False

No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.16
WARNING:conda_build.metadata:No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.16
INFO:conda_build.variants:Adding in variants from internal_defaults
Adding in variants from internal_defaults
Adding in variants from C:\ci\containers\00003ub53l8\tmp\build\80754af9\combined_recipe\conda_build_config.yaml
INFO:conda_build.variants:Adding in variants from C:\ci\containers\00003ub53l8\tmp\build\80754af9\combined_recipe\conda_build_config.yaml
INFO:conda_build.metadata:Attempting to finalize metadata for freetds
Attempting to finalize metadata for freetds
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... done
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... done
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... done
BUILD START: ['freetds-1.2.19-hcd4344a_0.tar.bz2']
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... done

## Package Plan ##

  environment location: C:\ci\freetds_1618227329495\_h_env


The following NEW packages will be INSTALLED:

    bzip2:           1.0.8-he774522_0      
    ca-certificates: 2021.1.19-haa95532_1  
    cmake:           3.19.6-h9ad04ae_0     
    jom:             1.1.2-h3bd88b9_0      
    libuv:           1.40.0-he774522_0     
    lz4-c:           1.9.3-h2bbff1b_0      
    openssl:         1.1.1k-h2bbff1b_0     
    perl:            5.26.2.1-h0c8e037_0   
    vc:              14.2-h21ff451_1       
    vs2015_runtime:  14.27.29016-h5e58377_2
    xz:              5.2.5-h62dcd97_0      
    zlib:            1.2.11-h62dcd97_4     
    zstd:            1.4.5-h04227a9_0      

Preparing transaction: ...working... done
Verifying transaction: ...working... done
Executing transaction: ...working... done
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... done
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... done

## Package Plan ##

  environment location: C:\ci\freetds_1618227329495\_build_env


The following NEW packages will be INSTALLED:

    vs2017_win-64: 19.16.27032.1-hb4161e2_3
    vswhere:       2.8.4-haa95532_0        

Preparing transaction: ...working... done
Verifying transaction: ...working... done
Executing transaction: ...working... done
Source cache directory is: C:\ci\containers\00003ub53l8\tmp\build\80754af9\output-source\src_cache
INFO:conda_build.source:Source cache directory is: C:\ci\containers\00003ub53l8\tmp\build\80754af9\output-source\src_cache
INFO conda_build.source:download_to_cache(44): Source cache directory is: C:\ci\containers\00003ub53l8\tmp\build\80754af9\output-source\src_cache
INFO:conda_build.source:Downloading source to cache: freetds-1.2.19_23985b05e0.tar.gz
INFO conda_build.source:download_to_cache(69): Downloading source to cache: freetds-1.2.19_23985b05e0.tar.gz
INFO:conda_build.source:Downloading http://www.freetds.org/files/stable/freetds-1.2.19.tar.bz2
INFO conda_build.source:download_to_cache(83): Downloading http://www.freetds.org/files/stable/freetds-1.2.19.tar.bz2
Downloading source to cache: freetds-1.2.19_23985b05e0.tar.gz
Downloading http://www.freetds.org/files/stable/freetds-1.2.19.tar.bz2
Success
INFO:conda_build.source:Success
INFO conda_build.source:download_to_cache(94): Success
Extracting download
source tree in: C:\ci\freetds_1618227329495\work
Rewriting env in output: {'BUILD_PREFIX': 'C:\\ci\\freetds_1618227329495\\_build_env',
 'PREFIX': 'C:\\ci\\freetds_1618227329495\\_h_env',
 'SRC_DIR': 'C:\\ci\\freetds_1618227329495\\work'}

(base) %SRC_DIR%>IF "" == "" (call %SRC_DIR%\build_env_setup.bat ) 

(base) %SRC_DIR%>set "CONDA_BUILD=1" 

(base) %SRC_DIR%>set "PYTHONNOUSERSITE=1" 

(base) %SRC_DIR%>set "CONDA_DEFAULT_ENV=%PREFIX%" 

(base) %SRC_DIR%>set "ARCH=64" 

(base) %SRC_DIR%>set "PREFIX=%PREFIX%" 

(base) %SRC_DIR%>set "BUILD_PREFIX=%BUILD_PREFIX%" 

(base) %SRC_DIR%>set "SYS_PREFIX=C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env" 

(base) %SRC_DIR%>set "SYS_PYTHON=C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env\python.exe" 

(base) %SRC_DIR%>set "SUBDIR=win-64" 

(base) %SRC_DIR%>set "build_platform=win-64" 

(base) %SRC_DIR%>set "SRC_DIR=%SRC_DIR%" 

(base) %SRC_DIR%>set "ROOT=C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env" 

(base) %SRC_DIR%>set "CONDA_PY=38" 

(base) %SRC_DIR%>set "PY3K=1" 

(base) %SRC_DIR%>set "PY_VER=3.8" 

(base) %SRC_DIR%>set "STDLIB_DIR=%PREFIX%\Lib" 

(base) %SRC_DIR%>set "SP_DIR=%PREFIX%\Lib\site-packages" 

(base) %SRC_DIR%>set "NPY_VER=1.16" 

(base) %SRC_DIR%>set "CONDA_NPY=1.16" 

(base) %SRC_DIR%>set "NPY_DISTUTILS_APPEND_FLAGS=1" 

(base) %SRC_DIR%>set "PERL_VER=5.26" 

(base) %SRC_DIR%>set "CONDA_PERL=5.26" 

(base) %SRC_DIR%>set "PERL=%PREFIX%\Library\bin\perl.exe" 

(base) %SRC_DIR%>set "LUA_VER=5" 

(base) %SRC_DIR%>set "CONDA_LUA=5" 

(base) %SRC_DIR%>set "R_VER=3.5" 

(base) %SRC_DIR%>set "CONDA_R=3.5" 

(base) %SRC_DIR%>set "PKG_NAME=freetds" 

(base) %SRC_DIR%>set "PKG_VERSION=1.2.19" 

(base) %SRC_DIR%>set "PKG_BUILDNUM=0" 

(base) %SRC_DIR%>set "PKG_BUILD_STRING=placeholder" 

(base) %SRC_DIR%>set "PKG_HASH=1234567" 

(base) %SRC_DIR%>set "RECIPE_DIR=C:\ci\containers\00003ub53l8\tmp\build\80754af9\combined_recipe" 

(base) %SRC_DIR%>set "CPU_COUNT=6" 

(base) %SRC_DIR%>set "SHLIB_EXT=.dll" 

(base) %SRC_DIR%>set "PATH=%BUILD_PREFIX%;%BUILD_PREFIX%\Library\mingw-w64\bin;%BUILD_PREFIX%\Library\usr\bin;;%BUILD_PREFIX%\Library\bin;%BUILD_PREFIX%\Scripts;%BUILD_PREFIX%\Scripts;%PREFIX%;%PREFIX%\Library\mingw-w64\bin;%PREFIX%\Library\usr\bin;;%PREFIX%\Library\bin;%PREFIX%\Scripts;%PREFIX%\Scripts;C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env;C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env\Library\mingw-w64\bin;C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env\Library\usr\bin;C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env\Library\bin;C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env\Scripts;C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env\bin;C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env\condabin;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.1\bin;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.1\libnvvp;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.0\bin;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.0\libnvvp;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.2\bin;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.2\libnvvp;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.0\bin;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.0\libnvvp;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v8.0\bin;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v8.0\libnvvp;C:\Program Files (x86)\Common Files\Intel\Shared Libraries\redist\intel64_win\mpirt;C:\Program Files (x86)\Common Files\Intel\Shared Libraries\redist\ia32_win\mpirt;C:\Program Files (x86)\Common Files\Intel\Shared Libraries\redist\intel64_win\compiler;C:\Program Files (x86)\Common Files\Intel\Shared Libraries\redist\ia32_win\compiler;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0;C:\Windows\System32\OpenSSH;c:\Program Files (x86)\Microsoft SQL Server\90\Tools\binn;C:\Users\builder\.dnx\bin;C:\Program Files\Microsoft DNX\Dnvm;C:\Program Files\Microsoft SQL Server\130\Tools\Binn;C:\Program Files\Mercurial;C:\Program Files\PuTTY;C:\curl\bin;C:\git\mingw64\bin;C:\Program Files (x86)\7-Zip;.;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\NVIDIA Corporation\Nsight Compute 2019.4.0;C:\Users\builder\AppData\Local\Microsoft\WindowsApps" 

(base) %SRC_DIR%>set "SCRIPTS=%PREFIX%\Scripts" 

(base) %SRC_DIR%>set "LIBRARY_PREFIX=%PREFIX%\Library" 

(base) %SRC_DIR%>set "LIBRARY_BIN=%PREFIX%\Library\bin" 

(base) %SRC_DIR%>set "LIBRARY_INC=%PREFIX%\Library\include" 

(base) %SRC_DIR%>set "LIBRARY_LIB=%PREFIX%\Library\lib" 

(base) %SRC_DIR%>set "CYGWIN_PREFIX=/cygdrive/c/ci/freetds_1618227329495/_h_env" 

(base) %SRC_DIR%>set "ALLUSERSPROFILE=C:\ProgramData" 

(base) %SRC_DIR%>set "APPDATA=C:\Users\builder\AppData\Roaming" 

(base) %SRC_DIR%>set "CommonProgramFiles=C:\Program Files\Common Files" 

(base) %SRC_DIR%>set "CommonProgramFiles(x86)=C:\Program Files (x86)\Common Files" 

(base) %SRC_DIR%>set "CommonProgramW6432=C:\Program Files\Common Files" 

(base) %SRC_DIR%>set "COMPUTERNAME=COLO-WWGPU-04" 

(base) %SRC_DIR%>set "ComSpec=C:\Windows\system32\cmd.exe" 

(base) %SRC_DIR%>set "LOCALAPPDATA=C:\Users\builder\AppData\Local" 

(base) %SRC_DIR%>set "NUMBER_OF_PROCESSORS=6" 

(base) %SRC_DIR%>set "PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC" 

(base) %SRC_DIR%>set "ProgramData=C:\ProgramData" 

(base) %SRC_DIR%>set "ProgramFiles=C:\Program Files" 

(base) %SRC_DIR%>set "ProgramFiles(x86)=C:\Program Files (x86)" 

(base) %SRC_DIR%>set "ProgramW6432=C:\Program Files" 

(base) %SRC_DIR%>set "PROMPT=(base) $P$G" 

(base) %SRC_DIR%>set "PSModulePath=C:\Program Files\WindowsPowerShell\Modules;C:\Windows\system32\WindowsPowerShell\v1.0\Modules" 

(base) %SRC_DIR%>set "PUBLIC=C:\Users\Public" 

(base) %SRC_DIR%>set "SystemDrive=C:" 

(base) %SRC_DIR%>set "SystemRoot=C:\Windows" 

(base) %SRC_DIR%>set "TEMP=C:\Users\builder\AppData\Local\Temp" 

(base) %SRC_DIR%>set "TMP=C:\Users\builder\AppData\Local\Temp" 

(base) %SRC_DIR%>set "USERDOMAIN=COLO-WWGPU-04" 

(base) %SRC_DIR%>set "USERNAME=builder" 

(base) %SRC_DIR%>set "USERPROFILE=C:\Users\builder" 

(base) %SRC_DIR%>set "windir=C:\Windows" 

(base) %SRC_DIR%>set "PROCESSOR_ARCHITECTURE=AMD64" 

(base) %SRC_DIR%>set "PROCESSOR_IDENTIFIER=Intel64 Family 6 Model 79 Stepping 1, GenuineIntel" 

(base) %SRC_DIR%>set "BUILD=amd64-pc-windows-19.0.0" 

(base) %SRC_DIR%>set "VS100COMNTOOLS=C:\Program Files (x86)\Microsoft Visual Studio 10.0\Common7\Tools\" 

(base) %SRC_DIR%>set "VS140COMNTOOLS=C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools\" 

(base) %SRC_DIR%>set "VS2017INSTALLDIR=C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional" 

(base) %SRC_DIR%>set "VS90COMNTOOLS=C:\Program Files (x86)\Microsoft Visual Studio 9.0\Common7\Tools\" 

(base) %SRC_DIR%>set "zip_keys=[['vc', 'c_compiler', 'cxx_compiler', 'fortran_compiler_version', 'python', 'geos']]" 

(base) %SRC_DIR%>set "pin_run_as_build=OrderedDict([('python', {'max_pin': 'x.x', 'min_pin': 'x.x'}), ('r-base', {'max_pin': 'x.x', 'min_pin': 'x.x'}), ('libboost', {'max_pin': 'x.x.x'})])" 

(base) %SRC_DIR%>set "extend_keys={'pin_run_as_build', 'extend_keys', 'ignore_version', 'ignore_build_only_deps'}" 

(base) %SRC_DIR%>set "ignore_build_only_deps={'numpy', 'python'}" 

(base) %SRC_DIR%>set "g2clib=1.6" 

(base) %SRC_DIR%>set "pixman=0.30" 

(base) %SRC_DIR%>set "hdfeos2=2.20" 

(base) %SRC_DIR%>set "libffi=3.2.1" 

(base) %SRC_DIR%>set "cran_mirror=https://mran.microsoft.com/snapshot/2018-01-01" 

(base) %SRC_DIR%>set "bzip2=1.0" 

(base) %SRC_DIR%>set "qtwebkit=true" 

(base) %SRC_DIR%>set "openjpeg=2.3" 

(base) %SRC_DIR%>set "serf=1.3.9" 

(base) %SRC_DIR%>set "m2w64_cxx_compiler=m2w64-toolchain" 

(base) %SRC_DIR%>set "hdfeos5=5.1" 

(base) %SRC_DIR%>set "cross_compiler_target_platform=win-64" 

(base) %SRC_DIR%>set "gnu=2.12.2" 

(base) %SRC_DIR%>set "proj=6.2.1" 

(base) %SRC_DIR%>set "r_version=3.5.0" 
CMake Error at CMakeLists.txt:85 (message):
  gperf not found!



(base) %SRC_DIR%>set "blas_impl=mkl" 

(base) %SRC_DIR%>set "expat=2.2" 

(base) %SRC_DIR%>set "boost_cpp=1.73" 

(base) %SRC_DIR%>set "libwebp=1.0.0" 

(base) %SRC_DIR%>set "clang_variant=clang" 

(base) %SRC_DIR%>set "libdap4=3.19" 

(base) %SRC_DIR%>set "hdf5=1.10.6" 

(base) %SRC_DIR%>set "proj4=5.2.0" 

(base) %SRC_DIR%>set "VERBOSE_AT=V=1" 

(base) %SRC_DIR%>set "fortran_compiler=intel-fortran" 

(base) %SRC_DIR%>set "libnetcdf=4.6" 

(base) %SRC_DIR%>set "macos_machine=x86_64-apple-darwin13.4.0" 

(base) %SRC_DIR%>set "libprotobuf=3.11.2" 

(base) %SRC_DIR%>set "rust_compiler=rust-gnu" 

(base) %SRC_DIR%>set "harfbuzz=2.4" 

(base) %SRC_DIR%>set "channel_targets=defaults" 

(base) %SRC_DIR%>set "boost=1.73" 

(base) %SRC_DIR%>set "macos_min_version=10.9" 

(base) %SRC_DIR%>set "libgsasl=1.8" 

(base) %SRC_DIR%>set "apr=1.6.3" 

(base) %SRC_DIR%>set "libgd=2.2.5" 

(base) %SRC_DIR%>set "target_platform=win-64" 

(base) %SRC_DIR%>set "xz=5" 

(base) %SRC_DIR%>set "libkml=1.3" 

(base) %SRC_DIR%>set "glib=2" 

(base) %SRC_DIR%>set "readline=8.0" 

(base) %SRC_DIR%>set "python_implementation=cpython" 

(base) %SRC_DIR%>set "lzo=2" 

(base) %SRC_DIR%>set "cairo=1.14" 

(base) %SRC_DIR%>set "gst_plugins_base=1.14" 

(base) %SRC_DIR%>set "cyrus_sasl=2.1.26" 

(base) %SRC_DIR%>set "noarch_python=python" 

(base) %SRC_DIR%>set "MACOSX_DEPLOYMENT_TARGET=10.9" 

(base) %SRC_DIR%>set "mkl=2019" 

(base) %SRC_DIR%>set "python_impl=cpython" 

(base) %SRC_DIR%>set "openblas=0.3.3" 

(base) %SRC_DIR%>set "libxml2=2.9" 

(base) %SRC_DIR%>set "rust_compiler_version=1.46.0" 

(base) %SRC_DIR%>set "tk=8.6" 

(base) %SRC_DIR%>set "freetype=2.10" 

(base) %SRC_DIR%>set "zlib=1.2" 

(base) %SRC_DIR%>set "dbus=1" 

(base) %SRC_DIR%>set "libgdal=3.0" 

(base) %SRC_DIR%>set "hdf4=4.2" 

(base) %SRC_DIR%>set "libxslt=1.1" 

(base) %SRC_DIR%>set "llvm_variant=llvm" 

(base) %SRC_DIR%>set "jpeg=9" 

(base) %SRC_DIR%>set "m2w64_c_compiler=m2w64-toolchain" 

(base) %SRC_DIR%>set "r_base=3.5" 

(base) %SRC_DIR%>set "cpu_optimization_target=nocona" 

(base) %SRC_DIR%>set "m2w64_fortran_compiler=m2w64-toolchain" 

(base) %SRC_DIR%>set "giflib=5" 

(base) %SRC_DIR%>set "libpng=1.6" 

(base) %SRC_DIR%>set "sqlite=3" 

(base) %SRC_DIR%>set "libtiff=4.1" 

(base) %SRC_DIR%>set "gstreamer=1.14" 

(base) %SRC_DIR%>set "mpfr=4" 

(base) %SRC_DIR%>set "gmp=6.1" 

(base) %SRC_DIR%>set "icu=58" 

(base) %SRC_DIR%>set "fontconfig=2.13" 

(base) %SRC_DIR%>set "r_implementation=mro-base" 

(base) %SRC_DIR%>set "VERBOSE_CM=VERBOSE=1" 

(base) %SRC_DIR%>set "openssl=1.1.1" 

(base) %SRC_DIR%>set "geos=3.8.0" 

(base) %SRC_DIR%>set "c_compiler=vs2017" 

(base) %SRC_DIR%>set "cxx_compiler=vs2017" 

(base) %SRC_DIR%>set "fortran_compiler_version=2019.0.0" 

(base) %SRC_DIR%>set "vc=14" 

(base) %SRC_DIR%>set "CONDA_BUILD_STATE=BUILD" 

(base) %SRC_DIR%>set "PIP_NO_BUILD_ISOLATION=False" 

(base) %SRC_DIR%>set "PIP_NO_DEPENDENCIES=True" 

(base) %SRC_DIR%>set "PIP_IGNORE_INSTALLED=True" 

(base) %SRC_DIR%>set "PIP_CACHE_DIR=C:\ci\freetds_1618227329495\pip_cache" 

(base) %SRC_DIR%>set "PIP_NO_INDEX=True" 

(base) %SRC_DIR%>set "INCLUDE=%PREFIX%\Library\include;" 

(base) %SRC_DIR%>set "LIB=%PREFIX%\Library\lib;" 

(base) %SRC_DIR%>call "C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env\Scripts\..\condabin\conda_hook.bat" 

(base) %SRC_DIR%>call "C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env\Scripts\..\condabin\conda.bat" activate "%PREFIX%" 

(%PREFIX%) %SRC_DIR%>call "C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env\Scripts\..\condabin\conda.bat" activate --stack "%BUILD_PREFIX%" 

%SRC_DIR%>SET DISTUTILS_USE_SDK=1 

%SRC_DIR%>SET MSSdk=1 

%SRC_DIR%>SET "VS_VERSION=15.0" 

%SRC_DIR%>SET "VS_MAJOR=15" 

%SRC_DIR%>SET "VS_YEAR=2017" 

%SRC_DIR%>set "MSYS2_ARG_CONV_EXCL=/AI;/AL;/OUT;/out" 

%SRC_DIR%>set "MSYS2_ENV_CONV_EXCL=CL" 

%SRC_DIR%>set "PY_VCRUNTIME_REDIST=%PREFIX%\bin\vcruntime140.dll" 

%SRC_DIR%>set "CXX=cl.exe" 

%SRC_DIR%>set "CC=cl.exe" 

%SRC_DIR%>CALL %BUILD_PREFIX%\etc\conda\activate.d\vs2017_get_vsinstall_dir.bat 
Windows SDK version found as: "10.0.18362.0"
**********************************************************************
** Visual Studio 2017 Developer Command Prompt v15.9.28
** Copyright (c) 2017 Microsoft Corporation
**********************************************************************
[vcvarsall.bat] Environment initialized for: 'x64'
-- The C compiler identification is MSVC 19.16.27043.0
-- The CXX compiler identification is MSVC 19.16.27043.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2017/Professional/VC/Tools/MSVC/14.16.27023/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2017/Professional/VC/Tools/MSVC/14.16.27023/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found Perl: C:/ci/freetds_1618227329495/_h_env/Library/bin/perl.exe (found version "5.26.2") 
-- Configuring incomplete, errors occurred!
See also "C:/ci/freetds_1618227329495/work/build/CMakeFiles/CMakeOutput.log".
Traceback (most recent call last):
  File "C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env\Scripts\conda-build-script.py", line 10, in <module>
    sys.exit(main())
  File "C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env\lib\site-packages\conda_build\cli\main_build.py", line 481, in main
    execute(sys.argv[1:])
  File "C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env\lib\site-packages\conda_build\cli\main_build.py", line 470, in execute
    outputs = api.build(args.recipe, post=args.post, test_run_post=args.test_run_post,
  File "C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env\lib\site-packages\conda_build\api.py", line 186, in build
    return build_tree(
  File "C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env\lib\site-packages\conda_build\build.py", line 3068, in build_tree
    packages_from_this = build(metadata, stats,
  File "C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env\lib\site-packages\conda_build\build.py", line 2159, in build
    windows.build(m, build_file, stats=build_stats, provision_only=provision_only)
  File "C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env\lib\site-packages\conda_build\windows.py", line 299, in build
    check_call_env(cmd, cwd=m.config.work_dir, stats=stats, rewrite_stdout_env=rewrite_env)
  File "C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env\lib\site-packages\conda_build\utils.py", line 412, in check_call_env
    return _func_defaulting_env_to_os_environ('call', *popenargs, **kwargs)
  File "C:\ci\containers\00003ub53l8\tmp\build\80754af9\build_env\lib\site-packages\conda_build\utils.py", line 392, in _func_defaulting_env_to_os_environ
    raise subprocess.CalledProcessError(proc.returncode, _args)
subprocess.CalledProcessError: Command '['cmd.exe', '/d', '/c', 'conda_build.bat']' returned non-zero exit status 1.
```

**Comment by freddy77** [comment]
Gperf can be easily installed in Windows too, see misc/appveyor.yml.

But it should not be required if the tarball was downloaded (it's required only if you need to manually generate some files, for instance building from git repository).

Trying a patch using Appveyor

**Comment by freddy77** [comment]
Should be fixed in 1.2.20.
